server/src/gateway/providers/object.action.service.ts

```diff
@@ -56,6 +57,7 @@ export class ObjectActionService {
       boardId,
       slideId,
       _id,
+      user._id as string,
```

Comment:
I think it would be better to inject in Gateway Module object.stats.service and use it for your purpose in object.action.service. In this approach we are breaking SRP principle by mixing object creation with stats